Cargo.toml

```
@@ -19,11 +19,14 @@ serde_json = "1.0.53"
 slog = "2.5.2"
 slog-async = "2.5.0"
 slog-term = "2.4.0"
-url = "1.7.2"
-tokio = { version = "0.2.21", features=["full"] }
+url = "2.1.1"
+tokio = { version = "0.2.22", features=["full"] }
 hyper = "0.13.6"
 native-tls = "0.2.4"
 tokio-native-tls = "0.1.0"
+hyper-tls = "0.4.3"
+ruma = "0.0.2"
+ruma-client = "0.4.0"
```

`ruma-client 0.4.0` re-exports the client api, so you don't need  the dependency on `ruma` with that. I think it would be good to remove it for now, and then use it when updating to `ruma-client 0.5.0` (or a prerelease of it) and tokio 1.0.

```suggestion
ruma-client = "0.4.0"
```